.github/workflows/pull_request.yml

```diff
@@ -103,6 +103,9 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - name: Setup and build wasm
+        uses: ./.github/actions/setup_build_wasm
+
```

Comment:
Sanity check: is the intention here to build the wasm targets and then run eslint on them in CI? Is there a specific pattern/issue that this is designed to catch (other than just making sure they build as part of a PR workflow)?

Reply:
I'm running lints that take into account types on the test code. The test code relies on types from the wasm build, so I have to build the wasm package prior to linting.